### PR TITLE
add: input html and CSS

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot (90)
Screenshot (91)

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
code looks clean to me ! just a little suggestion you can also add sript tag below body tag!reamaing looks good to me

File: Components/input.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ranitasahaa1230) [inline Components/input.html:7]: You can add author description, and page details in your meta tag.

**Review by ranitasahaa1230 — APPROVED** [review]
Rest looks good to me

File: Components/input.html (hunk 1 above)
Comment (aniketsingh13) [reply Components/input.html:7]: ok will do that

**Pull request merged**